hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -135,7 +135,7 @@ def test_get_calibration_from(self):
 class Test_quantification:
 
     def setUp(self):
-        s = EDSTEMSpectrum(np.ones([2, 1024]))
+        s = EDSTEMSpectrum(np.ones([2, 2, 1024]))
         energy_axis = s.axes_manager.signal_axes[0]
```

Comment:
Yes, so they're interchangeable as long as sub-pixel scanning is used (or sample at higher pixel sample and then binning down, which is mathematically identical). I was trying to avoid duplicating stuff that was already in the meta-data. But I see your point so maybe a probe_area addition to the metadata may be the simpler way to go?